utils/zerovec/src/ule/mod.rs

```diff
@@ -8,14 +8,19 @@
 mod chars;
 mod plain;
 mod string;
+mod vec;
 
 pub use chars::CharULE;
 pub use plain::PlainOldULE;
 
 /// Fixed-width, byte-aligned data that can be cast to and from a little-endian byte slice.
 ///
 /// "ULE" stands for "Unaligned little-endian"
-pub trait ULE
+///
+/// # Safety
+///
+/// See the safety invariant documented on [`Self::from_byte_slice_unchecked()`] to implement this trait.
+pub unsafe trait ULE
```

Comment:
See https://twitter.com/CAD97_/status/1436399266939932691 for why this needs to be unsafe